src/class-wp-plugin-loader.php

```diff
@@ -146,6 +146,24 @@ protected function load_plugins(): void {
 					$paths[] = "$folder/$sanitized_plugin/$sanitized_plugin.php";
 					$paths[] = "$folder/$sanitized_plugin/plugin.php";
 					$paths[] = "$folder/$sanitized_plugin.php";
+
+					if ( 0 === strpos( $sanitized_plugin, 'wordpress-' ) ) {
+						$paths[] = "$folder/" . substr( $sanitized_plugin, 10 ) . "/$sanitized_plugin.php";
+					} elseif ( 0 === strpos( $sanitized_plugin, 'wp-' ) ) {
+						$paths[] = "$folder/" . substr( $sanitized_plugin, 3 ) . "/$sanitized_plugin.php";
+					}
+
+					// Plugin-specific exceptions that don't follow the standard pattern.
+					$paths = array_merge(
+						$paths,
+						(array) match ( $sanitized_plugin ) {
```

Comment:
```suggestion
						match ( $sanitized_plugin ) {
```
Cast may be redundant here?